schema/commonmeta/commonmeta.schema.json

```diff
@@ -0,0 +1,679 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "$id": "https://commonmeta.org/commonmeta_v0.10.2.json",
+    "title": "Commonmeta v0.10.2",
+    "description": "JSON representation of the Commonmeta schema.",
+    "additionalProperties": false,
+    "definitions": {
+        "affiliations": {
+            "type": "array",
+            "items": {
+                "type": "object",
+                "properties": {
+                    "id": {
+                        "type": "string",
+                        "format": "uri"
+                    },
+                    "name": {
+                        "type": "string"
+                    }
+                },
+                "anyOf": [
+                    {
+                        "required": [
+                            "id"
+                        ]
+                    },
+                    {
+                        "required": [
+                            "name"
+                        ]
+                    }
+                ]
+            },
+            "uniqueItems": true
+        },
+        "contributorRole": {
+            "description": "The type of contribution made by a contributor",
+            "enum": [
+                "Author",
+                "Editor",
+                "Chair",
+                "Reviewer",
+                "ReviewAssistant",
+                "StatsReviewer",
+                "ReviewerExternal",
+                "Reader",
+                "Translator",
+                "ContactPerson",
+                "DataManager",
+                "Distributor",
+                "HostingInstitution",
+                "Producer",
+                "ProjectLeader",
+                "ProjectManager",
+                "ProjectMember",
+                "RegistrationAgency",
+                "RegistrationAuthority",
+                "RelatedPerson",
+                "ResearchGroup",
+                "RightsHolder",
+                "Researcher",
+                "Sponsor",
+                "WorkPackageLeader",
+                "Conceptualization",
+                "DataCuration",
+                "FormalAnalysis",
+                "FundingAcquisition",
+                "Investigation",
+                "Methodology",
+                "ProjectAdministration",
+                "Resources",
+                "Software",
+                "Supervision",
+                "Validation",
+                "Visualization",
+                "WritingOriginalDraft",
+                "WritingReviewEditing",
+                "Maintainer",
+                "Other"
+            ],
```

Comment:
Is this list CRediT +  DataCite contributor roles + Crossref? ORCID?

Reply:
CRediT, DataCite and Crossref. In a few cases there were duplicates, and then I picked the CRediT term. E.g. `Supervision` and `DataCuration`: https://github.com/front-matter/commonmeta-py/blob/b46a3e2986189b9ff5237bd75213486e73826847/commonmeta/constants.py#L458

Reply:
`Maintainer` comes from codemeta and is important for software.